packages/components/organisms/f-offers/src/store/offers.module.js

```diff
@@ -0,0 +1,122 @@
+import axios from 'axios';
+import jwtDecode from 'jwt-decode';
+import {
+    MUTATION_SET_AUTH_TOKEN,
+    MUTATION_SET_BRAZE_KEY,
+    MUTATION_SET_CUSTOMER_DATA,
+    ACTION_INITIALISE_OFFERS,
+    ACTION_HANDLE_CUSTOMER_DATA
+} from './types';
+
+/**
+ * Setup the config for axios
+ * @param authHeader
+ * @param timeout
+ * @returns {{headers: {Authorization, "Content-Type": string}, timeout}}
+ */
+const createAxiosConfig = (authHeader, timeout) => ({
+    headers: {
+        'Content-Type': 'application/json',
+        Authorization: authHeader
+    },
+    timeout
+});
+
+/**
+ * @function resolveCustomerDetails
+ * @param  {object} data  - Api response object.
+ * @param  {string} authToken - auth token.
+ */
+const resolveCustomerDetails = (data, authToken) => {
+    const tokenData = jwtDecode(authToken);
+    return {
+        phoneNumber: data?.customer?.phoneNumber ? data?.customer?.phoneNumber : tokenData?.phone_number,
```

Comment:
[Based on this](https://github.com/justeat/fozzie-components/blob/e6e58e52f8517c53bcfe255ba22c7c02b05dd3d3/packages/components/organisms/f-checkout/src/store/checkout.module.js#L57), do we also need to check `tokenData?.mobile_number` here?

Reply:
Yes you are correct, will make the addition